lib/utils/recommendation.ts

```diff
@@ -0,0 +1,21 @@
+import { interestsType } from "./getInterestOptions";
+
+
+const recommendation: Record<interestsType, string[]> = {
+  react: ["https://github.com/Skyscanner/backpack"],
```

Comment:
```suggestion
  react: ["Skyscanner/backpack"],
```
These should be repo full names instead of urls and the goal will eventually encouraging users to add these to insight pages.